recipes/onetbb/all/conanfile.py

```diff
@@ -46,6 +46,9 @@ def config_options(self):
             del self.options.shared
             del self.options.fPIC
 
+    def export_sources(self):
+        export_conandata_patches(self)
```

Comment:
You'll need to bump the `required_conan_version` up to `1.52.0` for `export_conandata_patches()`.